cmds/core/kexec/kexec_linux.go

```diff
@@ -56,15 +61,22 @@ func registerFlags() *options {
 	flag.StringVar(&o.initramfs, "initramfs", "", "Use file as the kernel's initial ramdisk")
 	flag.BoolVarP(&o.load, "load", "l", false, "Load the new kernel into the current kernel")
 	flag.BoolVarP(&o.exec, "exec", "e", false, "Execute a currently loaded kernel")
-	flag.BoolVarP(&o.debug, "debug", "d", false, "Print debug info")
+	flag.BoolVarP(&o.debug, "debug", "d", true, "Print debug info")
```

Comment:
Should this be true by default?

Reply:
for now ... this thing is not close to done :-)

Reply:
Done. 
 
false by default: https://github.com/u-root/u-root/pull/2317